db/migrate/20241112161300_remove_itf_datetime_from_saved_claims.rb

```diff
@@ -0,0 +1,5 @@
+class RemoveItfDatetimeFromSavedClaims < ActiveRecord::Migration[7.1]
+  def change
+    safety_assured { remove_column :saved_claims, :itf_datetime, :datetime }
```

Comment:
Can you add `if_exists: true`? This will avoid potential errors during the migration if the column didn't exist. No need to re-run the migration. 
```suggestion
    safety_assured { remove_column :saved_claims, :itf_datetime, :datetime, if_exists: true }
```